Services/RunnerService.cs

```diff
@@ -30,13 +30,15 @@ protected override async Task ExecuteAsync(CancellationToken stoppingToken)
 
                 if(!status)
                 {
-                    throw new Exception("Server is down");
+                    throw new Exception("Server is down.");
                 }
             }
-            catch(Exception e)
+            // A simplified, more dev-friendly error if we hit a rather standard error.
+            catch(HttpRequestException httpException)
             {
-                await _discordService.SetPresence(ActivityType.Watching, "server status: DOWN !");
-                Log.Error(e, "RunnerService");
+                await _discordService.SetPresence(ActivityType.Watching, "Server status: DOWN !");
```

Comment:
Makes more sense to have lower case server because the end result looks like this: "Watching server status: DOWN". Instead of "Watching Server status: DOWN"